dev-games/openscenegraph-openmw/openscenegraph-openmw-3.6_p20211017.ebuild

```diff
@@ -0,0 +1,167 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+LUA_COMPAT=( lua5-1 )
+
+WX_GTK_VER="3.0-gtk3"
+inherit cmake flag-o-matic lua-single wxwidgets
+
+MY_COMMIT="bbe61c3bc510a4f5bb4aea21cce506519c2d24e6"
+
+DESCRIPTION="OpenMW-specific fork of OpenSceneGraph"
+HOMEPAGE="https://github.com/OpenMW/osg"
+SRC_URI="https://github.com/OpenMW/osg/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/osg-${MY_COMMIT}"
+
+LICENSE="wxWinLL-3 LGPL-2.1"
+SLOT="0/162" # NOTE: CHECK WHEN BUMPING! Subslot is SOVERSION
```

Comment:
I was considering only libosg.so.162 for subslot, not libOpenThreads.so.21

Should the subslot be something like `0/162.21`?

Reply:
All the 162 and 21 are unchanged, what changed is these `rd`

Reply:
In my build there is nothing about `rd`

Reply:
ah, I see the issue, I tested it before switching EAPI, and didn't test as thoroughly after it; `rd` comes from  RelWithDebInfo

Reply:
Really? From a quick look, I feel it's related to this:
`SET(CMAKE_RELWITHDEBINFO_POSTFIX "rd" CACHE STRING "add a postfix, usually empty on windows")`

EAPI-8 cmake now defaults to relwithdebinfo, so it's probably related given this eapi bumps.

Reply:
...so rather than subslot bump, probably need to be worked around, or set different build type.

But if it doesn't happen for you, I'll check what might be different for me.

Reply:
> EAPI-8 cmake now defaults to relwithdebinfo

Yes, that's what I meant by

> rd comes from RelWithDebInfo

Reply:
Yeah I replied before I seen that message updated on my end.